build.sh

```diff
@@ -5,8 +5,24 @@
 
 set -e
 set -o pipefail
-pwd=$PWD
-chmod +x $pwd/submodules/klee/build.sh $pwd/submodules/Bear/build.sh $pwd/server/build.sh
-cd $pwd/submodules/klee && ./build.sh
-cd $pwd/submodules/Bear && ./build.sh
-cd $pwd/server && ./build.sh
+
+# Function arguments:
+#   - $1 - pwd to submodule with build.sh
+run_build_script() {
+  if [[ ! -f "$1/build.sh" ]]; then
+    echo "$1 doesn't exist."
+    exit 1
+  fi
+  chmod +x "$1/build.sh"
+  cd "$1" && ./build.sh
+}
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+BEAR_SRC="$SCRIPT_DIR/submodules/Bear"
+KLEE_SRC="$SCRIPT_DIR/submodules/klee"
+SERVER="$SCRIPT_DIR/server"
+
+run_build_script "$BEAR_SRC"
+run_build_script "$KLEE_SRC"
+run_build_script "$SERVER"
```

Comment:
I think, previous version was small, beautiful and clear and it is better than the current version.